### ISSUE TITLE
Can't remove liquidity from pool. M2 Devnet. 

### ISSUE THREAD

**Issue by valeria9131** [movementlabsxyz/bug-tracking#15]
BTC/USDT MOVE/USDT
![bug](https://github.com/movementlabsxyz/bug-tracking/assets/171065423/da11becf-05b0-499d-a3f3-685e2d758edc)

┆Issue is synchronized with this [Jira Task](https://movement.atlassian.net/browse/BUG-15) by [Unito](https://www.unito.io)

**Comment by andygolay** [comment]
Hi, thanks for reporting.

M2 had some outages, which we are aware of and working on fixing.

Have you mentioned this to Razor wallet support? I'm curious whether they have any insights on why this is happening.

**Comment by valeria9131** [comment]
sorry, the link in the discord is only to github, and I thought it would be better to write here - "Thank you for reporting any bugs you have found using Movement's infrastructure." 

M1 is working fine, I have added liquidity for all pairs, everything is ok. It seemed that M2 was working fine today too, I made a couple of swaps. But now it freezes and throws errors again.

I don't think they will share their thoughts on this issue with me, since I am just participating in testnet, but I will try to find out.))

Can I clarify for the future, I don't want to spam, what kind of bugs should I report? Or does it not make sense yet, since work on correction and optimization is constantly underway?
